src/Rules/Library/NameIsInformative.cs

```diff
@@ -45,6 +46,8 @@ protected override Condition CreateCondition()
         {
             return Name.NotNullOrEmpty & Name.NotWhiteSpace & BoundingRectangle.Valid
                 & ~Win32Framework
+                & ~PropertyConditions.ControlType.Group
```

Comment:
Please also add a unit test for each of the new exclusion cases. While you're touching the tests, one of the existing tests (FrameworkWind32_DoesNotMatch) is misnamed (should be "Win32" instead of "Wind32"). Would you mind fixing the typo?